### PR TITLE
Milestone 2 - User Interface

### PR DESCRIPTION
**Pull request by rsnazario**
- Added the main interactions, which are:
  - Ask if the user wants to play the game
  - Ask for the names of the players
  - Ask for moves of the players
  - Print Board after each move
  - Warning if the entry is invalid
  - Provide feedback based on input by user
  - Added dummy game logic for demonstration

### CODE REVIEW

File: bin/main [hunk 1]
```diff
@@ -1,2 +1,126 @@
...
+  when 8
+    board[2][2] = param
+  when 9
+    board[2][4] = param
+  end
+end
+
+def test_play?(play_made)
+  play_made >= 1 && play_made <= 9 ? true : false
+end
```
Comment (mosesogwo) [inline bin/main:77]: Take note that a move that has already been made before is also invalid.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**HI @rsnazario, @Elbie-em. Good work:+1:. In addition to what you have, your game mock should reflect the following :fast_forward:**

1. After each move, the game should check if it is a winning move or a draw and inform the players.

2. The game loop shouldn't break arbitrarily. It should break only when there is a  winning move or a draw.

_**Kindly make these changes and submit another code review request.**_

**Any question? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by rsnazario** [comment]
Applied messages to show if:
  - the movement is invalid (out of possible range)
  - the movement has already been done for the other player
  - result of Match (which player won or draw match)
  - question for another round

**Review by Forison — APPROVED** [review]
## Status: Approved💚
# 🙋‍♂️ team,
Good job on implementing the changes raised earlier. The game does not break arbitrarily. 
See you at the next milestone.
#### Happy coding💻
Addo

**Pull request merged**